saleor/graphql/schema.graphql

```diff
@@ -1448,6 +1448,33 @@ enum ProductOrderField {
   DATE
 }
 
+enum ProductTaxRate {
```

Comment:
We already have the `TaxRateType` in the schema. The difference is that `TaxRateType` is implemented explicitly by us, while this one is generated by Graphene after introducing `choices` in `tax_rate` field (I guess). We should only keep one of them in the schema. I think the `TaxRateType` would be easier to drop but the question is how can we use the `ProductTaxRate` in other places.

Reply:
Is it good now?

Reply:
Looks good now! We're changing the type here and since we keep the same name of the field, we cannot deprecate it. This means that we have to pass this PR to @dominik-zeglen to fix references to `taxRate` field in the Dashboard 2.0.